Ungate branch state page

andyhuang91 opened the pull request:
This change updates the urls for everyone to link to the new branch state page. There is no more check for beta vs non-beta users. Links have been updated everywhere so the only way to reach the old page is by clicking on the `Build History` link.

I updated the wording a little for the new feature alert. Here's what it currently says
<img width="1104" alt="screen shot 2016-12-08 at 5 45 51 pm" src="https://cloud.githubusercontent.com/assets/4141884/21030814/76bc3ba0-bd6e-11e6-82be-a4f597788d51.png">

cc @markhazlewood @gchomatas @jonathanwgoodwin

andyhuang91 committed:
Update all branch links to point to branch state page

In preparation of releasing the branch state page as the main page to all users.
Also cleans up the generation of BlazarUI paths.

andyhuang91 committed:
Repurpose branch state beta feature alert message to new feature alert message

Simply updating the text and using the same local storage variable to avoid showing
current beta users another new feature message. This alert will be removed in a
week or two after releasing to everyone.

Jw0x47 commented:
The tone / voice on that message is 👍 🔥 🎉

markhazlewood commented:
Can you add some emphasis (`<strong>`) on "you'll find the latest build information for each module". 

It will help the most important bit pop out for people who will only be skimming this (which you should assume is **everyone**)

andyhuang91 committed:
Update new branch state page notification

- Add emphasis you'll find the latest build information
- Create link to open up feedback form
- Update color for info alerts

andyhuang91 commented:
Applying mark's updates to bold some of the text and add a link to expand the feedback form from the alert.

<img width="1099" alt="screen shot 2016-12-09 at 11 35 21 am" src="https://cloud.githubusercontent.com/assets/4141884/21056457/b3e28756-be03-11e6-8bd7-a559ca002073.png">

markhazlewood commented:
⚓️
